packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -336,8 +338,9 @@ export class StripeHelper {
     return this.stripe.subscriptions.create(
       {
         customer: customerId,
-        items: [{ price: priceId, tax_rates: taxRates }],
+        items: [{ price: priceId }],
         expand: ['latest_invoice.payment_intent'],
+        default_tax_rates: taxRates,
```

Comment:
Unrelated fix?

Reply:
It's a closely related fix, yes. The tax rates were in the wrong spot.